src/components/Chat/InputMessageV2.tsx

```diff
@@ -191,7 +191,7 @@
         gif_path: gifPath
       }
     ];
-    onSendButtonClicked('Sent GIF 🎆🌆🌉', resultUrls);
+    onSendButtonClicked('Sent GIF 🏞️', resultUrls);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the message string from 'Sent GIF 🎆🌆🌉' to 'Sent GIF 🏞️' doesn't seem to have any functional impact. As with the previous comments, it's important to ensure that these changes are consistent across all instances where this message is used to maintain a uniform user experience.
```diff
-    onSendButtonClicked('Sent GIF 🎆🌆🌉', resultUrls);
+    onSendButtonClicked('Sent GIF 🏞️', resultUrls);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->